widgetastic_manageiq.py

```diff
@@ -878,3 +878,144 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class Stepper(Widget):
+    """ A CFME Stepper Control
+
+        stepper = Stepper(locator='//div[contains(@class, "timeline-stepper")]')
+        stepper.inrease()
+    """
+    def __init__(self, parent, locator, logger=None):
+        super(Stepper, self).__init__(parent=parent, logger=logger)
+
+        self._stepper_control = self.browser.element(locator, parent=parent)
+
+        self.minus_button = Button(self, '-')
+        self.plus_button = Button(self, '+')
+        self.value_field = Input(self, locator='.//input[contains(@class, '
```

Comment:
Do you think you could try converting this to a View and subsequently define the buttons and input ON the class as with regular view? You will then need to call out to View.__init__ instead of Widget.__init__ but the params remain the same.